### DISCUSSION TITLE
User-Agent Override?

### DISCUSSION THREAD

**Discussion by cmwright-wcrclocal** [grafana/grafana-infinity-datasource#227] [Q&A]
Hi, We have been using Grafana and Infinity to collect Web application Status information for some of the Web Applications that we use, this enables us to check the status of multiple web apps quickly. Recently one of these web application has been blocking the user-agents of none standard browsers. I have read here that we can override the user-agent for Infinity? https://github.com/yesoreyeram/grafana-infinity-datasource/blob/f5ef999451d4bb948bdf562637ee328cb2006ec6/website/src/wiki/url.md however this does not seem to be working for us. Is there another way to do it, or what is the exact string that the headers should look like?

Thanks,

**Comment by yesoreyeram** [comment]
Have you tried latest version? According to https://github.com/yesoreyeram/grafana-infinity-datasource/blob/76bd45294b58abfc86db119b03b81548a5dd586a/pkg/infinity/client.go#L115 it should override..

**Reply by cmwright-wcrclocal** [reply]
Yeah, we are running the latest version, but has not made any difference.

**Reply by cmwright-wcrclocal** [reply]
Upon further Investigation, if I use curl -H to add a header "User-Agent: Chrome/79" then its blocked, but if I use the command for adding --user-agent 'Chrome/79' then it works. I believe Cloudflare is able to ignore spoofed user agents via header input.

**Reply by yesoreyeram** [reply]
Hmm.. not sure if anything needs to fixed from plugin side..

**Reply by KarloDerEchte** [reply]
I am encountering the same issue, setting the User-Agent in wget & curl works. But when i set the header in the plugin it doesnt.
